lib/copy_gauge_extended.cu

```diff
@@ -65,7 +65,7 @@ namespace quda {
 #else
         errorQuda("QUDA_RECONSTRUCT=%d does not enable reconstruct-8", QUDA_RECONSTRUCT);
 #endif
-#ifdef GPU_STAGGERED_DIRAC
+#if defined(GPU_STAGGERED_DIRAC) || defined(BUILD_OPENQCD_INTERFACE)
```

Comment:
`defined(BUILD_OPENQCD_INTERFACE)` should be replaced by `defined(QUDA_QCD_PLUS_QED)` as appropriate